docs/authorization-events.md

```diff
@@ -0,0 +1,31 @@
+# Authorization events
+
+Luigi provides life-cycle events that can be triggered by Luigi internally, or by authorization providers.
+Events are part of the **auth** configuration object and require to be functions, they can be executed asynchronous.
+
+By returning `false` value in those functions, the default behaviour can be disabled and lifecycle execution stops with the function. Be aware that this might lead to blank pages in logged out state, since most of the time there is a redirect to either a logout, login or main page.
+
```

Comment:
I would put the code sample right after the second sentence. So we have:

`Events are part of the **auth** configuration object and require to be functions, they can be executed asynchronous.`
Then maybe an introductory line for the sample.
`An example events configuration looks as follows:`
<code sample here>

Then I would take the first two sentences from the ## Events section so that we have the description of the first and second parameter (what can it be). At the end, we can have some info on disabling the default behavior by returning false :)  

So basically the ##Events section would only have the description of functions, since the other, more descriptive information would go with the code sample.

What do you think about it?